src/modules/trajectory_generation/interface/package.xml

```diff
@@ -10,16 +10,21 @@
   <license>Tensor Robotics</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
```

Comment:
Remove. Since it is a dev tool we should not be including it as a node in main package. I tested by removing all python dependencies in package.xml and CMakeLists.txt and __init__.py. Everything works fine.